community/modules/scheduler/schedmd-slurm-gcp-v6-controller/scripts/cleanup_compute.sh

```diff
@@ -52,7 +52,7 @@ while true; do
 done
 
 echo "Deleting resource policies"
-policies_filter="name:${cluster_name}-*"
+policies_filter="name:${cluster_name}-slurmgcp-managed-*"
```

Comment:
What is the longest allowed name for a resource policy? My concern is that this has implicit impact on customers with "long" cluster or nodeset names.